src/styles/theme/_variables.scss

```diff
@@ -24,6 +24,7 @@ $color-form-field: #EEEEEE;
 $color-mine-shaft: #3A3A3A;
 $color-skeleton: #E9EEEE;
 $color-cadre-formulaire: #EEF0F7;
+$color-mentor: #18753C;
```

Comment:
petit tip pour nommer les variables : https://chir.ag/projects/name-that-color/#18753C
comme ça tu auras un nom de variable qui ne sera pas lié à cette utilisation